src/Console_App_Calculator/MathUtils.cs

```diff
@@ -0,0 +1,57 @@
+﻿// <copyright file="MathUtils.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+namespace ConsoleAppCalculator;
+
+/// <summary>
+/// MathUtils contains the methods for arithmetic operations.
+/// </summary>
+internal static class MathUtils
+{
+    /// <summary>
+    /// Adds Integer1 to Integer2
+    /// </summary>
+    /// <param name="a">Integer1</param>
+    /// <param name="b">Integer2</param>
+    public static void Add(int a, int b)
+    {
+        Console.WriteLine($"Sum of {a} and {b} is {a + b}");
+    }
+
+    /// <summary>
+    /// Divides Integer1 by Integer2
+    /// </summary>
+    /// <param name="a">Integer1</param>
+    /// <param name="b">Integer2</param>
+    public static void Divide(int a, int b)
+    {
+        try
```

Comment:
Check for zero instead of using a try catch exception statement to improve performance.

Reply:
Changed as required